src/ImageService.Data/ImageProjectRepository.cs

```diff
@@ -1,78 +1,63 @@
-﻿using LT.DigitalOffice.ImageService.Data.Interfaces;
-using LT.DigitalOffice.ImageService.Data.Provider;
-using LT.DigitalOffice.ImageService.Models.Db;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using LT.DigitalOffice.ImageService.Data.Interfaces;
+using LT.DigitalOffice.ImageService.Data.Provider;
+using LT.DigitalOffice.ImageService.Models.Db;
+using Microsoft.Data.SqlClient;
 
 namespace LT.DigitalOffice.ImageService.Data
 {
-    public class ImageProjectRepository : IImageProjectRepository
-    {
-        private readonly IDataProvider _provider;
+  public class ImageProjectRepository : IImageProjectRepository
+  {
+    private readonly IDataProvider _provider;
 
-        public ImageProjectRepository(IDataProvider provider)
-        {
-            _provider = provider;
-        }
-
-        public List<Guid> Create(List<DbImageProject> imagesProjects)
-        {
-            if (imagesProjects.Contains(null))
-            {
-                return null;
-            }
-
-            _provider.ImagesProjects.AddRange(imagesProjects);
-            _provider.Save();
-
-            return imagesProjects.Select(x => x.Id).ToList();
-        }
+    public ImageProjectRepository(IDataProvider provider)
+    {
+      _provider = provider;
+    }
 
-        public bool Remove(List<Guid> imageIds)
-        {
-            if (imageIds == null)
-            {
-                return false;
-            }
+    public List<Guid> Create(List<DbImageProject> imagesProjects)
+    {
+      if (imagesProjects.Contains(null))
+      {
+        return null;
+      }
 
-            List<DbImageProject> imagesProjects = _provider.ImagesProjects
-                .Where(x => imageIds.Contains(x.Id) || (x.ParentId != null && imageIds.Contains((Guid)x.ParentId)))
-                .ToList();
+      _provider.ImagesProjects.AddRange(imagesProjects);
+      _provider.Save();
 
-            if (imagesProjects == null)
-            {
-                return false;
-            }
+      return imagesProjects.Select(x => x.Id).ToList();
+    }
 
-            List<Guid> parentIds = new();
+    public bool Remove(List<Guid> imageIds)
+    {
+      if (imageIds == null)
+      {
+        return false;
+      }
 
-            foreach (DbImageProject imageProject in imagesProjects)
-            {
-                if (imageProject.ParentId != null
-                    && imageProject.Id != imageProject.ParentId
-                    && !imageIds.Contains((Guid)imageProject.ParentId))
-                {
-                    parentIds.Add((Guid)imageProject.ParentId);
-                }
-            }
+      SqlCommand command = new();
```

Comment:
Why do you create it, if you don't use it. Here you use it for saving string. I guess it is strange

Reply:
done